plugins/push/frontend/public/javascripts/countly.views.js

```diff
@@ -1,161 +1,434 @@
-'use strict';
-
-/* jshint undef: true, unused: true */
-/* globals app, $, countlyAuth, countlyGlobal, components, countlyCommon, countlySegmentation, countlyUserdata, CountlyHelpers, jQuery, countlyManagementView, Backbone */
-
-var featureName = 'push';
-
-app.addAppManagementView('push', jQuery.i18n.map['push.plugin-title'], countlyManagementView.extend({
-    initialize: function() {
-        this.plugin = 'push';
-        this.templatePath = '/push/templates/push.html';
-        this.resetTemplateData();
-    },
-
-    resetTemplateData: function() {
-        var c = this.config();
-        if (c.i && c.i._id) {
-            this.templateData = {
-                i: {
-                    _id: c.i._id,
-                    type: c.i.type,
-                    key: c.i.key,
-                    team: c.i.team,
-                    bundle: c.i.bundle,
-                    help: c.i.type === 'apn_universal' && c.i._id ? '<i class="fa fa-check-circle"></i>' + jQuery.i18n.map['mgmt-plugins.push.uploaded.p12'] : c.i.type === 'apn_token' ? '<i class="fa fa-check-circle"></i>' + jQuery.i18n.map['mgmt-plugins.push.uploaded.p8'] : ''
-                    // help: '<a href="' + countlyCommon.API_URL + '/i/pushes/download/' + c.i._id + '?api_key=' + countlyGlobal.member.api_key + '">' + jQuery.i18n.map['mgmt-plugins.push.uploaded'] + '</a>. ' + (c.i.type === 'apn_universal' ? (jQuery.i18n.map['mgmt-plugins.push.uploaded.bundle'] + ' ' + c.i.bundle) : '')
-                }
+/* global countlyVue,app,CV,countlyPushNotification,CountlyHelpers,jQuery,countlyManagementView,countlyCommon,$,countlyGlobal,countlyAuth,countlySegmentation,countlyUserdata, components,Backbone*/
+(function() {
+
+    var AUTOMATIC_PUSH_NOTIFICATION_STATUS_FILTER_OPTIONS = [
+        {label: CV.i18n("push-notification.table-filter-scheduled"), value: countlyPushNotification.service.StatusEnum.SCHEDULED},
+        {label: CV.i18n("push-notification.table-filter-all"), value: countlyPushNotification.service.StatusEnum.ALL},
+        {label: CV.i18n("push-notification.table-filter-sent"), value: countlyPushNotification.service.StatusEnum.SENT},
+        {label: CV.i18n("push-notification.table-filter-sending"), value: countlyPushNotification.service.StatusEnum.SENDING},
+        {label: CV.i18n("push-notification.table-filter-aborted"), value: countlyPushNotification.service.StatusEnum.ABORTED},
+        {label: CV.i18n("push-notification.table-filter-failed"), value: countlyPushNotification.service.StatusEnum.FAILED},
+        {label: CV.i18n("push-notification.table-filter-stopped"), value: countlyPushNotification.service.StatusEnum.STOPPED}
+    ];
+    var TRANSACTIONAL_PUSH_NOTIFICATION_STATUS_FILTER_OPTIONS = AUTOMATIC_PUSH_NOTIFICATION_STATUS_FILTER_OPTIONS;
```

Comment:
That's what I call variable naming. Now we're talking 😄

Reply:
hah, you know that I am always open to suggestions right :slightly_smiling_face:

Reply:
No I mean it. It's Java-esque

Reply:
ok, now you're making me feel bad about it :laughing: . cheers.